docs/spec/lcd/readme.md

```diff
@@ -0,0 +1,47 @@
+# Overview
+
+These documents describe the Light Client Daemon, commonly referred to as LCD. The LCD is split into
+two separate components. 
+The first component is generic for any Tendermint based application. It 
+handles the security and connectivity aspects of following the header chain and verifying proofs
+from full nodes against locally trusted state roots. Furthermore it exposes exactly the same API as
+any Tendermint Core node.
+The second component is specific for the Cosmos Hub (Gaiad). It works through the query endpoint on
+Tendermint Core and exposes the application specific functionality, which can be arbitrary. All 
+queries against the application state have to go through the query endpoint. The advantage of the
+query endpoint is that it can verify the proofs that the application returns.
+
+An application developer that wants to build an third party application for the Cosmos Hub (or any
+other zone) should build it against it's canonical API. That API is a combination of multiple parts.
```

Comment:
The canonical API you are speaking about here is the `lcd` correct? If so this paragraph should make clear that the `lcd` is the implementation of this.